lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1247,8 +1247,12 @@ export class UserAgentApplication {
     if (this.getPostLogoutRedirectUri()) {
       logout = "post_logout_redirect_uri=" + encodeURIComponent(this.getPostLogoutRedirectUri());
     }
-    const urlNavigate = this.authority + "oauth2/v2.0/logout?" + logout;
-    this.promptUser(urlNavigate);
+    this.authorityInstance.resolveEndpointsAsync().then(a => {
```

Comment:
can we use a more descriptive param name than a?

Reply:
Of course I've used 'authority' as it is of type 'Authority'. We could change it to something in respect to the async method name (e.g. endpoints)?